app/assets/javascripts/discourse/app/templates/modal/create-invite.hbs

```diff
@@ -55,7 +55,7 @@
     {{/if}}
 
     {{#if currentUser.staff}}
-      <p>
+      <p id="show-advanced">
```

Comment:
Can we make this a class instead? I worry that `#show-advanced` is too global an id otherwise and might conflict with some other HTML. Additionally you could name it `#invite-show-advanced`